By-laws: Allow out-of-season self-nominations (#572)
ghc-proposals/ghc-proposals: committee.rst

The nomination and voting process is kept private, by using direct
email to committee members, not the mailing list. 

> Additionally, anyone may self-nominate themselves anytime and trigger a vote. 

Reviewer: How, concretely, does a volunteer self-nominates out of season?

Author: Good point. By email to the secretary? (Public email would be bad.)

Reviewer: An email to the secretary is probably the right way, indeed.. We just need to make it explicit.

Author: ```suggestion
Additionally, anyone may self-nominate themselves anytime (by email to the secretary). 
```

> Such "out-of-season" appointments are at the discretion of the committee, and

Reviewer: I'm not sure I'm following what the intent is here. The first sentence in the paragraph says that an out-of-season nomination triggers a vote. The second seems to mean that the committee decides whether it votes or not.

There is more than one way this can be interpreted. One is that, in case of an out-of-season nomination, the committee decides whether or not to hold a vote (that being said, how do they do that? are they holding a vote about the vote? I guess that the difference here would be that this is a decision about the relevance of an extra seat rather than on the particular person). Another decision is that the committee's vote (triggered by the first sentence) has, as an option, not to elect the nominee; though it would be trivial, since there are only two options in such an out-of-season vote (elect or not).

Author: Maybe it would be better to say “at the discretion of the secretary” or “the chair”? Some simple mechanism to avoid abuse of the process?

Reviewer: Fair enough. If the trigger is an email to the secretary, they can also be the filter. It would make sense.

Author: ```suggestion
Such "out-of-season" appointments are at the discretion of the secretary, and
```
